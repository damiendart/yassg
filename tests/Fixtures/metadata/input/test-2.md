---
#
#
#
twigTemplate: '.templates/test.html.twig'
wrapperTag: 'div'
---
<h1>This is another test!</h1>
