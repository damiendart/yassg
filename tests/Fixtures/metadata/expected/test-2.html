<div><h1>This is another test!</h1></div>
